lib/voxpupuli/test/rake/symlinks.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+require 'pathspec'
+
+module Voxpupuli
+  module Test
+    module Rake
+      # Helpers for validating symlinks.
+      class Symlinks
+        DEFAULT_IGNORED = [
+          '/.git/',
+          '/.bundle/',
```

Comment:
I was looking at this and if we make this relative then is the `child.basename.to_s` check redundant? I'd expect them to match in `ignored?` then.
```suggestion
          '.git',
          '.bundle',
```